step2-05/README.md

```diff
@@ -38,7 +40,7 @@ There is a single dispatcher. It simply informs the store of all the actions tha
 
 ### Reducers
 
-Redux uses **reducers** to manage state changes. This name comes from the "reducer" function passed to `Array.reduce()`.
+Redux uses reducers to manage state changes. This name comes from the "reducer" function passed to `Array.reduce()`.
```

Comment:
Question that I thought of while reading through this (which might come up from others too)... To truly be a "reducer," must a function work on the whole state, or is it still a reducer if it only handles part of the state?

After going through the materials more, I think the answer is that it's okay to have separate reducers to handle subsets of the state, but the only reducer that you pass to `createStore` is a main one which ties all the little reducers together.